netaddr_test.go

```diff
@@ -2726,3 +2726,19 @@ func TestNoAllocs(t *testing.T) {
 		sinkIPPrefix = panicPfxOK(a.Prefix())
 	})
 }
+
+func TestIPPortString(t *testing.T) {
+	tests := []struct {
+		ipp  IPPort
+		want string
+	}{
+		{IPPort{}, "invalid IPPort"},
+		{IPPortFrom(IP{}, 80), "invalid IPPort"},
```

Comment:
add a test to lock in behavior of non-zero IP and zero port.

Reply:
We have a fair amount of those in TestParseIPPort. I'll name this function to make it clearer that it is about invalid IPPorts.